validator/client/log_test.go

```diff
@@ -0,0 +1,67 @@
+package client
+
+import (
+	"testing"
+
+	field_params "github.com/prysmaticlabs/prysm/v5/config/fieldparams"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+	ethpb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/testing/assert"
+	"github.com/prysmaticlabs/prysm/v5/testing/require"
+	"github.com/prysmaticlabs/prysm/v5/testing/util"
+	logTest "github.com/sirupsen/logrus/hooks/test"
+)
+
+func TestLogSubmittedAtts(t *testing.T) {
```

Comment:
can we have one more testcase here where we'll have more than one committee index as part of `committeeIndices` array log?